src/libs/common.go

```diff
@@ -565,3 +586,50 @@ func ConvertStrToUnixTime(strTime string) int64 {
 	t, _ := time.Parse(TimeFormSimple, strTime)
 	return t.UTC().Unix()
 }
+
+//ConvertArrayToString - Convert Array of string to String
+func ConvertArrayToString(arr []string) string {
+	var str string
+	for _, label := range arr {
+		if !strings.HasPrefix(label, "k8s:io.cilium.") {
+			if !strings.HasPrefix(label, "k8s:io.kubernetes.") {
+				tstr := strings.TrimPrefix(label, "k8s:")
+				if str != "" {
+					str += constants.COMMA
+				}
+				str += tstr
+			}
+		}
+	}
+	return str
+
+}
+
+//ConvertStringToArray - Convert String to Array of string
+func ConvertStringToArray(str string) []string {
+	return strings.Split(str, ",")
+}
+
+func ConvertFilterString(filter []string) string {
+	var query string
+	//Create the filter query
+	for i, value := range filter {
+		query = query + constants.BACK_SLASH + fmt.Sprint(value) + constants.BACK_SLASH
```

Comment:
I think having constants define is not helping here! It was better to use literals directly.
```suggestion
		query = query + "\" + fmt.Sprint(value) + "\"
```

Reply:
This is just an existing implementation porting from observability to discovery-engine. 
Will handle this .